client/web/reporter/src/mixins/report.js

```diff
@@ -95,14 +97,12 @@ export default {
     handleClone (report) {
       this.processing = true
       this.processingClone = true
-      const { handle, meta, sources, blocks, scenarios, labels } = report
+      const clonedReport = this.report.clone()
+      const { handle: reportHandle, meta, sources, blocks, scenarios, labels } = clonedReport
       meta.name = `${meta.name} (${this.$t('general:cloneSuffix')})`
+      const handle = reportHandle ? `${report.handle}_${this.$t('general:cloneSuffix')}` : ''
```

Comment:
I know i said to add the clone to the handle, but it just occurred to me that could raise an error if you clone the same report twice (they'll both have same handle)
So its honestly better to not have a handle at all.